TYPO3.Flow/Migrations/Code/Version20151113161300.php

```diff
@@ -0,0 +1,161 @@
+<?php
+namespace TYPO3\Flow\Core\Migrations;
+
+/*
+ * This file is part of the TYPO3.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Configuration\ConfigurationManager;
+use TYPO3\Flow\Security\RequestPattern\ControllerObjectName;
+use TYPO3\Flow\Security\RequestPattern\CsrfProtection;
+use TYPO3\Flow\Security\RequestPattern\Host;
+use TYPO3\Flow\Security\RequestPattern\Ip;
+use TYPO3\Flow\Security\RequestPattern\Uri;
+
+/**
+ * Adjust "Settings.yaml" to new "requestPattern" and "firewall" syntax (see FLOW-412)
+ */
+class Version20151113161300 extends AbstractMigration
```

Comment:
This will adjust request patterns like this:
![image](https://cloud.githubusercontent.com/assets/307571/11151602/4fcb5d06-8a2e-11e5-9f3f-ba934e1339f7.png)
and firewall filters like:
![image](https://cloud.githubusercontent.com/assets/307571/11151608/594a171e-8a2e-11e5-98cd-495c75410694.png)


I'm not perfectly sure whether using the classname for the index is a good idea!?